package.json

```diff
@@ -427,6 +427,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "%asciidoc.useWorkspaceRoot.desc%"
+				},
+				"asciidoc.antora.enableAntoraSupport": {
+					"type": "boolean",
+					"default": false,
+					"description": "%asciidoc.antora.desc%"
```

Comment:
you can use `markdownDescription` instead of `description` to use Markdown syntax in the description